### PR TITLE
Add: Video like, Page for Liked videos, few transitions

### PR DESCRIPTION
**Pull request by saurabhchirde**
Feature Added

1. Video like and removing like
2. Page for Liked Videos
3. Some transitions

video-like

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -14,7 +14,7 @@
   width: 100%;
   height: 120vh;
   position: fixed;
-  z-index: 1;
+  z-index: 3;
```
Comment (dev-ashishsethi) [inline src/App.css:17]: use variable for z-index

File: src/Components/Cards/VerticalCard.jsx [hunk 2]
```diff
@@ -56,9 +96,15 @@ export const VerticalCard = ({ videoDetail }) => {
...
     } else {
-      setWatchlaterButton("far fa-clock");
+      setWatchlaterButton("far fa-clock icon-inactive");
     }
-  }, [watchlater, videoDetail._id, setWatchlaterButton]);
+
+    if (likes.findIndex((el) => el._id === videoDetail._id) !== -1) {
+      setLikeButton("fas fa-thumbs-up");
+    } else {
+      setLikeButton("far fa-thumbs-up icon-inactive");
```
Comment (dev-ashishsethi) [inline src/Components/Cards/VerticalCard.jsx:105]: using variables for classnames would be good i think

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

File: src/Context/AxiosCalls/AxiosCallProvider.jsx [hunk 3]
```diff
@@ -107,16 +107,21 @@ const AxiosCallProvider = ({ children }) => {
 
   // un-like video
   const unLikeVideoOnServer = async (likeConfig) => {
-    const { url, headers, item } = likeConfig;
+    const {
```
Comment (m26unkwn) [inline src/Context/AxiosCalls/AxiosCallProvider.jsx:110]: code quality is good and loves the changes to the alert component.

**Review by m26unkwn — APPROVED** [review]
1. Naming convention is good.
2.  Love the changes
3. Card is looking great.

**Pull request merged**